floris/wind_data.py

```diff
@@ -776,7 +779,10 @@ def plot(
                 )
 
         # Configure the plot
-        ax.legend(reversed(rects), ws_bins, **legend_kwargs)
+        # ax.legend(reversed(rects), ws_bins, **legend_kwargs)
+        cb_ws = ax.figure.colorbar(sm_ws, ax=ax, **legend_kwargs)
+        ax.figure.tight_layout()
+        cb_ws.outline.set_visible(False)
```

Comment:
I actually didn't mean to leave the `cb_ws.outline.set_visible(False)`, which I added to conform to my personal style preferences for a paper. if you want to match standard numpy viz, you might want to remove that line.